components/aws/sagemaker/TrainingJob/samples/mnist-kmeans-classification/README.md

```diff
@@ -0,0 +1,82 @@
+# MNIST Classification with KMeans
+
+This sample aims to showcase the usage of v1 components and the new v2 training job component together in a Kubeflow pipeline workflow. The `mnist-classification-pipeline.py` sample runs a pipeline to train a classficiation model using Kmeans with MNIST dataset on SageMaker. This example was taken from an existing [SageMaker example](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/1P_kmeans_highlevel/kmeans_mnist.ipynb) and modified to work with the Amazon SageMaker Components for Kubeflow Pipelines.
+
+## Prerequisites 
+
+1. Make sure you meet all the configuration prerequisites *including the optional step* for using the v2 components from sagemaker/TrainingJob README (https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker/TrainingJob). 
```

Comment:
nit: Would a relative link be better here, maybe the long links can become hyperlinks for more consistency.

Reply:
Don't think theres any benefit to using relative links, none of the other samples are using them. If others think it should be changed i will go ahead and do so.